headers/xdp/parsing_helpers.h

```diff
@@ -101,6 +102,39 @@ static __always_inline int parse_ethhdr(struct hdr_cursor *nh, void *data_end,
 	return h_proto; /* network-byte-order */
 }
 
+static __always_inline int skip_ip6hdrext(struct hdr_cursor *nh,
+					  void *data_end,
+					  __u8 currhdr)
```

Comment:
Please find a better variable name than 'currhdr'. Maybe 'next_proto'?

Reply:
I decided to name it that way to mirror the "nexthdr" field of the headers struct. Also, I think that calling it protocol is not accurate, since the headers are part of the IPv6 protocol. But I don't really feel strongly about the naming.

Reply:
But it's not the 'current header' as the name applies. Rather it's the 'current next header' - which obviously is not a very good variable name. But just using 'currhdr' is confusing; if you can come up with something better, feel free! :)